Observer/CaptureObserver.php

```diff
@@ -108,18 +108,19 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         $payment          = $observer['payment'];
         $invoice          = $observer['invoice'];
         $orderIncrementId = $invoice->getOrder()->getIncrementId();
+        $paymentType      = $payment->getAdditionalInformation('payment_type');
         $orderObject      = $this->order->loadByIncrementId($orderIncrementId);
         $storeCode        = $invoice->getStore()->getCode();
         if (in_array($payment->getMethod(), SystemConfig::getTerminalCodes())) {
             //Create orderlines from order items
             $orderLines = $this->processInvoiceOrderLines($invoice);
             //Send request for payment refund
-            $this->sendInvoiceRequest($invoice, $orderLines, $orderObject, $payment, $storeCode);
+            $this->sendInvoiceRequest($paymentType, $invoice, $orderLines, $orderObject, $payment, $storeCode);
         }
     }
 
     /**
-     * @param $invoice
+     * @param null|object $invoice
```

Comment:
If `null` was given here the application would crash on line 129, the correct object appears to be `Magento\Sales\Model\Order\Invoice`:
```suggestion
     * @param Magento\Sales\Model\Order\Invoice $invoice
```